web/src/app/donate/DonateComponent.js

```diff
@@ -120,6 +122,27 @@ class DonateComponent extends Component {
     });
   }
 
+  getGameOptions() {
+    return this.props.topGames.map((game, idx) => ({
+      id: idx,
+      img: game.box_art_url,
+      name: game.name,
+    }));
+  }
+
+  renderOption(option, props, index) {
+    const width = 50;
+    const height = 75;
+    const url = option.img.replace('{width}', width).replace('{height}', height);
+
+    return (
+      <div>
+        <img className={styles.gameThumbnail} src={url} alt={option.name} width={width} height={height} />
```

Comment:
I typically try and keep javascript free from css. What is the benefit for adding height and width directly to the image here and not adding it to the css className?

Reply:
Width/Height here isn't css, its using the HTML width and height attributes. This is typically a better was to keep images the same size for layout purposes.